docs/deployment/digital-ocean-apps.mdx

```diff
@@ -237,6 +237,324 @@ Done! Application deployed and can be accessed by provided domain.
 
 ![Deployed application](/images/deployed-application.png)
 
+## Github Actions with prebuilt Container Images (Optional)
```

Comment:
use mintlify section for reusing this documentation block

Also add it to page "DO Apps (IaC)"